README.md

```diff
@@ -111,42 +111,53 @@ To install:
  
 ### CMake Options
 
-If you run the `cmake` command by hand, there are many options you can
-set with "-D". For example, the option to build the RADOS Gateway is
-defaulted to ON. To build without the RADOS Gateway:
+The `-D` flag can be used with the `cmake` command to speed up the process of
+building Ceph and to customize the build.
+
+#### Building without RADOS Gateway
+
+The RADOS Gateway is built by default. To build Ceph without the RADOS Gateway,
+run a command of the following form:
 
 	cmake -DWITH_RADOSGW=OFF [path to top-level ceph directory]
 
-Another example below is building with debugging and alternate locations 
-for a couple of external dependencies:
+#### Building with debugging and arbitrary dependency locations 
+
+Run a command of the following form to build Ceph with debugging and alternate
+locations for some external dependencies:
 
 	cmake -DCMAKE_INSTALL_PREFIX=/opt/ceph -DCMAKE_C_FLAGS="-Og -g3 -gdwarf-4" \
 	..
 
 Ceph has several bundled dependencies such as Boost, RocksDB and Arrow. By
-default, cmake will build these bundled dependencies from source instead of
-using libraries that are already installed on the system. You can opt-in to
-using these system libraries, provided they meet the minimum version required
-by Ceph, with cmake options like `WITH_SYSTEM_BOOST`:
+default, cmake builds these bundled dependencies from source instead of using
```

Comment:
Above we backtick `cmake`

Reply:
I was on the fence about it, but if your fresh eyes expect it, then we do it.

Reply:
Either way, be consistent.